### ISSUE TITLE
Crash running move-dir during initial sync

### ISSUE THREAD

**Issue by red-scott** [samschott/maestral#594]
**Describe the bug**

`move-dir` run during initial sync causes a crash

I'm normally not the kind of maniac who would attempt such a thing, but Maestral's onboarding is confusing and resulted in the situation.

1. I have a large Dropbox, 60GB, it's important Maestral uses my existing local dropbox copy, which it says it can do, but if I was prompted for a "merge", I never saw it.
2. Maestral transparently created a new Dropbox directory with the (Maestral) suffix. Once I realized it hadn't actually merged my files as promised and I was going to have to wait for 60GB of download, I realized I needed to move the directory, otherwise I'd be operating on my old Dropbox directory that is no longer being updated.
3. moved old `~/Dropbox` out of the way
4. ran `maestral move-dir Dropbox`

The onboarding should look for an existing dropbox directory as the default IMO.

**To Reproduce**

1. start syncing large repository
2. run `maestral move-dir Dropbox`
3. command hangs for several minutes
4. then:

```
> maestral move-dir Dropbox
Traceback (most recent call last):
  File "Contents/Resources/Support/Python/Resources/lib/python3.10/runpy.py", line 196, in _run_module_as_main
  File "Contents/Resources/Support/Python/Resources/lib/python3.10/runpy.py", line 86, in _run_code
  File "Contents/Resources/app/maestral_cocoa/__main__.py", line 41, in <module>
  File "Contents/Resources/app/maestral_cocoa/__main__.py", line 36, in freeze_support_cli
  File "Contents/Resources/app_packages/click/core.py", line 1128, in __call__
  File "Contents/Resources/app_packages/click/core.py", line 1053, in main
  File "Contents/Resources/app_packages/click/core.py", line 1659, in invoke
  File "Contents/Resources/app_packages/click/core.py", line 1395, in invoke
  File "Contents/Resources/app_packages/click/core.py", line 754, in invoke
  File "Contents/Resources/app_packages/maestral/cli.py", line 1464, in move_dir
  File "Contents/Resources/app_packages/Pyro5/client.py", line 476, in __call__
  File "Contents/Resources/app_packages/Pyro5/client.py", line 243, in _pyroInvoke
OSError: [Errno 30] Read-only file system
 +--- This exception occured remotely (Pyro) - Remote traceback:
 | Traceback (most recent call last):
 |   File "Contents/Resources/Support/Python/Resources/lib/python3.10/shutil.py", line 805, in move
 | OSError: [Errno 18] Cross-device link: '/Users/red/Dropbox (Maestral)' -> '/Dropbox'
 | 
 | During handling of the above exception, another exception occurred:
 | 
 | Traceback (most recent call last):
 |   File "Contents/Resources/app_packages/Pyro5/server.py", line 466, in handleRequest
 |   File "Contents/Resources/app_packages/maestral/main.py", line 1206, in move_dropbox_directory
 |   File "Contents/Resources/Support/Python/Resources/lib/python3.10/shutil.py", line 821, in move
 |   File "Contents/Resources/Support/Python/Resources/lib/python3.10/shutil.py", line 556, in copytree
 |   File "Contents/Resources/Support/Python/Resources/lib/python3.10/shutil.py", line 457, in _copytree
 |   File "Contents/Resources/Support/Python/Resources/lib/python3.10/os.py", line 225, in makedirs
 | OSError: [Errno 30] Read-only file system: '/Dropbox'
 +--- End of remote traceback
```

**Expected behaviour**

**System:**

 - Maestral version:
 - Python version: 
 - OS: [e.g. Ubuntu]
 - Desktop environment: [e.g. Gnome 3.32]
 - PyQt version (for Linux GUI):

**Additional context**

MacBook Air (M1, 2020)
macOS 12.1

**Comment by red-scott** [comment]
`maestral` does not appear to attempt to create the specified directory for the `move-dir` command, and if it's not there, that seems to cause the crash

**Comment by samschott** [comment]
Thanks for the report! It appears that Maestral attempts to create the directory, however not at "~/Dropbox" but at "/Dropbox". This fails since a typical user account cannot write to the root directory. I'll fix this. For now, try specifying the absolute path.

Also note that `maestral move-dir` will work if the destination directory already exists.

> The onboarding should look for an existing dropbox directory as the default IMO.

I've considered this but I'm not sure if this is a good idea. Once synced with Maestral, some metadata may be lost that is used by Dropbox internally for its own client. This might make switching back to the official client annoying since it will likely perform a reindexing of all local files. Another consideration is that accidentally running both at the same time and pointing to the same folder could have bad results (e.g., infinite sync loops).

**Comment by red-scott** [comment]
Thanks, Sam! I appreciate the incredibly fast work.

I think users with large Dropbox accounts and slower internet connections (or just bad wifi) are probably a common enough use case it's worth being aggressive in finding their existing Dropbox directory and trying to use that.

Conflicts with normal Dropbox running are obviously also a problem and ideally you'd identify if Dropbox is currently running when you try to merge with a dropbox directory. But I can see potential issues there as well, so fair enough.

My presumption was the program would make an effort to find and use my Dropbox directory, especially when I saw text saying merging was an option and I specified my directory as my existing directory. The fact that `maestral` ignored what I specified and started downloading a HUGE amount of files was extremely jarring and threw me.

As a result, I was a bit annoyed when I wrote the bug so I apologize if my report was a bit short. Now that I have everything up and running I'm quite pleased with it and I think you for all your effort!
